src/modules/party-external-rating/party-external-rating.controller.ts

```diff
@@ -0,0 +1,48 @@
+import { Controller, Get, Inject, Param } from '@nestjs/common';
+import { ApiInternalServerErrorResponse, ApiNotFoundResponse, ApiOkResponse, ApiOperation, ApiParam } from '@nestjs/swagger';
+
+import { GetPartyExternalRatingResponse, GetPartyExternalRatingsResponse } from './dto/get-party-external-ratings-response.dto';
+import { PARTY_EXTERNAL_RATINGS_PROVIDER_SYMBOL } from './party-external-rating.module-definition';
+import { PartyExternalRatingsProvider } from './party-external-ratings.provider';
+
+@Controller()
+export class PartyExternalRatingController {
+  constructor(@Inject(PARTY_EXTERNAL_RATINGS_PROVIDER_SYMBOL) private readonly partyExternalRatingsProvider: PartyExternalRatingsProvider) {}
+
+  @Get('/party/:partyIdentifier/external-rating')
+  @ApiOperation({ summary: 'Get all external ratings for a party.' })
+  @ApiOkResponse({
+    description: 'The external ratings for the party have been successfully retrieved.',
+    type: GetPartyExternalRatingResponse,
+    isArray: true,
+  })
+  @ApiNotFoundResponse({
+    description: 'The specified party, or the external ratings for that party, were not found.',
+  })
+  @ApiInternalServerErrorResponse({
+    description: 'An internal server error has occurred.',
+  })
+  async getExternalRatingsForParty(
+    @Param('partyIdentifier')
+    partyIdentifier: string,
```

Comment:
do we expect the `partyIdentifier` to have a specific format - I'm asking in case we need a DTO for it? if not, I'm happy for it to be string

Reply:
I'll check with @avaitonis

Reply:
Ok cool. Normally I would say to validate it, but...
Let's leave it as it is for now as it may break some of the existing functionalities in other systems.